### DISCUSSION TITLE
Add New Wallet (SentienceID) to Stable RainbowKit

### DISCUSSION THREAD

**Discussion by awidearray** [rainbow-me/rainbowkit#1904] [Q&A] [answered]
Can't seem to find a route to add our new wallet to rainbow kit. Can you point me in the direction? 
thanks!
ios / android / chrome ext would love to be native in the list!

**Comment by awidearray** [comment]
Wallet is SentienceId 
is basically a Metamask fork

**Comment by 0xmkh** [comment]
@futjrnaut Which RainbowKit version are you trying to use with your new wallet ? Here is an example of [coinbaseWallet](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/wallets/walletConnectors/coinbaseWallet/coinbaseWallet.ts). Feel free to check out our docs [here](https://www.rainbowkit.com/docs/custom-wallets).

Let me know if this helps 🙏

**Reply by awidearray** [reply]
we're finally live
is there a way to get this added as an option in the list of wallets? @magiziz thanks
https://chromewebstore.google.com/detail/fadgogmokmnbjlfnnhdombhkbekdcflm

**Reply by 0xmkh** [reply, marked as answer]
Yeah i can add it to RainbowKit. Just a quick question, are you supporting any EVM chains by any chance ?

**Reply by awidearray** [reply]
yes currently mainnet bsc poly will add base and other rpcs as necessary in the future :)

**Reply by awidearray** [reply]
any update here? can close once approved thanks

**Reply by 0xmkh** [reply]
Hey there. Apologies for the delay, we'll give you a sound as soon as possible when this is done 👍

**Reply by awidearray** [reply]
trust you're well. 
we've got eip6963 in is there an update to wallet connect that needs their newest json? 
https://github.com/SentienceQuest/SentienceID-Ext/releases/tag/v1.0.3
Please let me know we're happy to discuss how to be in the upper list with resourcing if needed. 
appreciate the hard work and look forward to joining the ranks in the extended search.

**Reply by 0xmkh** [reply]
Hey. That's great that the wallet supports EIP-6963, i'll have a look at this as soon as possible.
